incubator-tools/cds_dataset_creator/cds_dataset_creator.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>Shoebox? Is this an industry standard term?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
No, it's not. It's a term the customer uses to describe a composite PDF file containing multiple documents. We will update the term to a more generalized naming convention.